Eng translation com.1

Gawidev opened the pull request:
Translation tweaks and general eng language modifications for better readability.

Gawidev committed:
Eng translation com.1

Translation tweaks and general eng language modifications for better readability.

Creta5164 commented:
Thank you for contribution!  
I'll review it later. (2:39 AM here)

Creta5164 commented:
* In this review, I can submit a change request for parts where the text is cut beyond the area.  
For this I need to check on the actual game, so it may take some time!

Creta5164 commented on en/Map001.json:
```diff
@@ -5,10 +5,10 @@
             "\\C[3]LShift\\C[0] : \\C[14](Hold)\\C[0] Run",
-            "The goal of the game is to grow Legendary tree safely.\\!",
-            "You will need \\C[3]water\\C[0] to grow a tree.\\!",
+            "The goal of the game is to grow Legendary tree and keep it safe.\\!",
```
![image](https://user-images.githubusercontent.com/13521406/151901339-ee56a802-1d00-478c-8625-f0e6f389ca2a.png)  
You should move the end of this part to the space of next line or shorten this.

Creta5164 commented on en/Map001.json:
```diff
@@ -28,29 +28,29 @@
-            "Let's give it moderately well.",
-            "If Legendary tree has \\C[14]a state can able to grow up\\C[0],\\.\\.",
+            "Let's give it a healthy serving.",
+            "If The Legendary tree has \\C[14]a state allows it to grow bigger\\C[0],\\.\\.",
```
![image](https://user-images.githubusercontent.com/13521406/151901998-c4d9ac70-1691-42c4-bba1-2c6c9dd7b096.png)  
You should put line break `\n` or shorten this.

Creta5164 commented on en/CommonEvents.json:
```diff
@@ -2,25 +2,25 @@
-        "Hero hears from him about how to break her curse.",
-        "\"It is said that there was a tree that had only been",
-        "passed down as a legend in the Mt. [Legendary]...\"",
+        "The curse placed upon the Princess was a very terrible one",
```
![image](https://user-images.githubusercontent.com/13521406/151900658-fba4d937-57e9-482a-a57d-49ad99ff23b3.png)  
You should put line break `\n` or shorten this.

Creta5164 commented on en/CommonEvents.json:
```diff
@@ -2,25 +2,25 @@
-        "passed down as a legend in the Mt. [Legendary]...\"",
+        "The curse placed upon the Princess was a very terrible one",
+        "A curse that gradually transforms one into a monster...!",
+        "The Hero seeks the sage's advice to break the Princess's curse,",
```
![image](https://user-images.githubusercontent.com/13521406/151900704-3a6707c4-d407-49e6-a0b4-c9d0366c1c0a.png)  
You should move the end of this part to the space of next line or shorten this.

Creta5164 commented on en/CommonEvents.json:
```diff
@@ -35,30 +35,30 @@
         "\\$You gave him all the \\G you had.",
         "Damn,\\. you don't have enough\\G\\C[3]\\V[9]\\C[0],\\.\\.",
-        "...I'll let you pass this time cause your efforts...",
+        "...I'll let you pass this time because of your cooperation...",
```
![image](https://user-images.githubusercontent.com/13521406/151900944-c20eb6e6-a6c6-4ced-a734-d71aa51a2404.png)  
You should put line break `\n` or shorten this.

Creta5164 commented on en/CommonEvents.json:
```diff
@@ -35,30 +35,30 @@
     ],
     "26": [
-        "I have everything except not have stuff,\ntake a look...",
+        "I have everything other than not having stuff,\ntake a look...",
```
![image](https://user-images.githubusercontent.com/13521406/151901064-400b32e5-b634-40d8-96dd-62b02b1d72cc.png)  
You should move line break `\n` or shorten this.

Creta5164 commented on en/CommonEvents.json:
```diff
@@ -105,22 +105,22 @@
+        "Ye-, Yes!\\!",
         "I have turned into a monster because the curse, but",
-        "I'm the princess who was waiting for hero's salvation...",
+        "I'm the princess who was waiting for the hero's salvation...",
```
![image](https://user-images.githubusercontent.com/13521406/151901220-b837acac-5c03-494e-bb67-3afa0e890c01.png)  
You should put line break `\n` or shorten this.

Creta5164 committed:
Merge branch 'main' into pr/1

Creta5164 committed:
Merge branch 'main' into pr/1

Creta5164 reviewed (changes requested):
Perfect! I can see that almost all sentences are neatly arranged.  
Now, you only need to correct some of the overflowing sentences.
Then I can approve this PR.

Once again, thanks for the contribution!

Gawidev committed:
Eng translation tweaks

Took the liberty to better fit the translations on top of fixing the overflowing lines.

Creta5164 reviewed (approved):
Ok, I've checked that everything is displayed well.  
Thanks for your contribution!
